valpas-web/package.json

```diff
@@ -27,6 +27,10 @@
   },
   "license": "EUPL-1.1",
   "homepage": "https://github.com/Opetushallitus/koski#readme",
+  "browserslist": [
+    "last 3 Chrome versions",
+    "last 3 Firefox versions"
+  ],
```

Comment:
Oliko meillä aiemmin browserslist kohdetta asetettu lainkaan? Tai onko sillä ollut jokin oletusarvo?

Reply:
Ei ole ollut aiemmin. En enää näin lomien muista tarkalleen miksi tämän jouduin lisäämään. Ajattelin vaihtaa tämän tilalle nyt kuitenkin dokumentaation suositteleman arvon `"defaults"`.

Reply:
Koska `defaults` toi mukanaan tuen kaiken maailman esoteerisille kännykkäselaimille, kiristin tuettujen selainten listan viimeisimpään kolmeen versioon Firefoxille, Chromelle ja Edgelle, jotta ei tarvitse turhaan polyfilleja kaataa bundleen mukaan.